src/clr-angular/button/button-loading/loading-button.spec.ts

```diff
@@ -134,7 +134,8 @@ describe('Loading Buttons', () => {
     `,
 })
 class TestLoadingButtonComponent {
-  @ViewChild(ClrLoadingButton) loadingButtonInstance: ClrLoadingButton;
+  @ViewChild(ClrLoadingButton, { static: false })
```

Comment:
Are most queries in specs considered dynamic because of the way the tests run? This seems like it is static based on how I understood the migration guide. Or does selecting a class directly instead of a template var also make it dynamic?